code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm

```diff
@@ -22,3 +22,28 @@
 		return
 	if(. == CONSCIOUS && layer != initial(layer))
 		layer = MOB_LAYER
+
+/mob/living/carbon/xenomorph/roony 
+	caste_base_type = /mob/living/carbon/xenomorph/roony
+	name = "Roony"
```

Comment:
Again is there a particular reason this isnt just a child of runner

Reply:
Yes, much easier like that, else i have to do a shit ton of code for no benefits